app/src/views/private/components/file-preview.vue

```diff
@@ -37,6 +40,18 @@ const props = withDefaults(defineProps<Props>(), { width: undefined, height: und
 
 const imgError = ref(false);
 
+const imageError = ref<string | null>(null);
+
+async function onErrorHandle() {
+	if (!props.src) return;
+	try {
+		await api.get(props.src);
+	} catch (err: any) {
+		imgError.value = true;
+		imageError.value = err;
+	}
+}
```

Comment:
This error handling is fetching the image fresh to extract the error, even though the `v-image` that throws the error already emits the error. Lets reuse the error emitted by `v-image`

Reply:
I completely agree with you @rijkvanzanten to reuse code. But error emitted by `v-image` has responseType `ArrayBuffer` which `v-error` component would not be able to handle it. Therefore result might look like attached in screenshot. 

![image](https://user-images.githubusercontent.com/93135924/213366556-68a8210b-69c7-4fbc-aaf6-67c50614f514.png)
![image](https://user-images.githubusercontent.com/93135924/213366015-1ddeda2c-f33b-4785-8a43-ccaf3845248d.png)

### Reason of making new request
In order to address this responseType `ArrayBuffer`, I had made new request. 

### Solution
- 1st option  
We can handle converting Responsetype to String in `v-image` Component. 
![image](https://user-images.githubusercontent.com/93135924/213369075-92a7b4da-8555-40e2-b7a5-72503e1a1a6b.png)

- 2nd option 
We can handle this conversion of  responseType in `v-error` Component.

Reply:
Let me know for any updates.